client/components/event-create-page.js

```diff
@@ -11,6 +12,9 @@ export const EventFormPage = props => (
   </div>
 )
 const mapDispatchToProps = dispatch => ({
-  onSubmit: newEvent => dispatch(postEvent(newEvent))
+  onSubmit: newEvent => {
+    dispatch(postEvent(newEvent))
+    history.push('/home')
```

Comment:
You should only do this in the thunk, after the DB updates, which avoids a redirect/reload of events before the new event is in the DB.